cove_iati/management/commands/iati_cli.py

```diff
@@ -17,19 +16,28 @@ def add_arguments(self, parser):
                             'Org-ids prefixes')
         super(Command, self).add_arguments(parser)
 
-    def handle(self, file, *args, **options):
-        super(Command, self).handle(file, *args, **options)
+    def handle(self, files, *args, **options):
+        super(Command, self).handle(files, *args, **options)
+        stream = options.get('stream')
         openag = options.get('openag')
         orgids = options.get('orgids')
 
         try:
-            result = iati_json_output(self.output_dir, file, openag=openag, orgids=orgids)
+            if stream:
+                for file in files:
+                    if os.path.isdir(file):
+                        self.stdout.write('Skipping %s directory ' % str(file))
+                    else:
+                        result = iati_json_output(self.output_dir, file, openag=openag,
+                                                  orgids=orgids)
```

Comment:
I think there should not be any need for an output directory when using streaming.
I think a temporary directory should be made instead, that is automatically deleted afterwards.  We do this for the IATI api here.
https://github.com/OpenDataServices/cove/blob/master/cove_iati/views.py#L100

Reply:
We also should add the filename to the ouput JSON also.